tests/kw_time_and_date_test.sh

```diff
@@ -28,13 +28,22 @@ function test_sec_to_format()
 function test_get_today_info()
 {
   local today
+  local today_double_check
 
   today=$(date +%Y/%m/%d)
   formated_today=$(get_today_info '+%Y/%m/%d')
   assert_equals_helper 'Today info did not match' "$LINENO" "$today" "$formated_today"
 
-  formated_today=$(get_today_info)
   today=$(date)
+  formated_today=$(get_today_info)
+  today_double_check=$(date)
+
+  if [[ "$today" != "$today_double_check" ]]; then
+    today=$(date)
+    formated_today=$(get_today_info)
+    today_double_check=$(date)
```

Comment:
This second assignment is not needed, as this value will not be used past this point.

Reply:
Thanks to the comment about line 44, I do not know how we forget this line there. But I am not sure if I understood what you said about the problem with this solution. Would you suggest just comparing the variables `today` and `formatted_today` instead of creating a new variable?